.github/workflows/release.yml

```diff
@@ -31,3 +31,4 @@ jobs:
         head: releases/${{ steps.date.outputs.date }}
         base: release
         title: Release ${{ steps.date.outputs.date }}
+        team_reviewers: release
```

Comment:
Should we create a group first?
I don't see it in https://github.com/orgs/neondatabase/teams

Reply:
https://github.com/orgs/neondatabase/teams/release

Here it is